src/Twee/School/School assistant.twee

```diff
@@ -436,7 +436,7 @@
 		You drop to your knees and, with a trembling hand, reach for the zipper on the leader's pants. He eagerly helps you get his pants down, and you slowly pull his underwear down too, letting his thick cock pop free.
 	<<endif>>\
 	
-	@@.hero;Thick, but not that huge.@@ With relief, you glance at it for a moment, transfixed, wondering how it will taste. Your cheeks blush a bit at that last thought. @@.hero;I shouldn't find it attractive. I'm not a faggot. I like girls.@@ 
+	@@.hero;Thick, but not that huge.@@ With relief, you glance at it for a moment, transfixed, wondering how it will taste. Your cheeks blush a bit at that last thought. @@.hero;I shouldn't find it attractive. <<if $kinkAllow.cursing>>I'm not a faggot. <</if>>I like girls.@@ 
```

Comment:
```suggestion
	@@.hero;Thick, but not that huge.@@ With relief, you glance at it for a moment, transfixed, wondering how it will taste. Your cheeks blush a bit at that last thought. @@.hero;I shouldn't find it attractive. <<if $kinkAllow.curse>>I'm not a faggot. <</if>>I like girls.@@ 
```